lib/components/Atom/Calendar/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect } from "react";
+import clsx from "clsx";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import "./date-picker.scss";
+import {
+    LabelPairedChevronRightSmFillIcon,
+    LabelPairedChevronLeftSmFillIcon,
+} from "@deriv/quill-icons";
+import { Text } from "@components/Typography";
+
+interface DatePicker
+    extends Omit<
+        React.ComponentProps<typeof Calendar>,
+        "onChange" | "activeStartDate"
+    > {
+    isWidthFixed?: boolean;
+    optionsConfig?: Record<string, string>;
+    onChange?: (
+        value: Value,
+        event?: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    ) => void;
+    onFormattedDate?: (value: string) => void;
+    startDate?: Date;
+    wrapperClassName?: string;
+}
+
+type ValuePiece = Date | null;
+
+type Value = ValuePiece | [ValuePiece, ValuePiece];
+
+export const DatePicker = ({
+    startDate,
+    allowPartialRange = false,
+    calendarType,
+    className,
+    defaultValue,
+    defaultView,
+    isWidthFixed = true,
```

Comment:
it's up to you but what do you think about naming this prop `hasFixedWidth` ? :)

Reply:
Sure, sounds better that current prop) Will push the fix